apps/directory/src/views/BiobankReport.vue

```diff
@@ -160,9 +160,12 @@ export default {
           : {};
       });
 
-    return { settingsStore, biobanksStore, qualitiesStore, biobank };
+    return { settingsStore, biobanksStore, qualitiesStore, biobank, route };
   },
   methods: {
+    urlWithProps(props: string) {
```

Comment:
props is a bit of a confusing name, since it is also a vue key-word. Maybe something like "queryProperties"?